polytope/polytope.py

```diff
@@ -442,6 +468,134 @@ def text(self, txt, ax=None, color='black'):
         _plot_text(self, txt, ax, color)
 
 
+def _translate(polyreg, d):
+    """Translate a polyreg by a vector in place. Does not return a copy.
+
+    @type d: 1d array
+    @param d: The translation vector.
+
+    @type polyreg: L{Polytope} or L{Region}
+    @param polyreg: The polytope or region to be translated.
+    """
+
+    if isinstance(polyreg, Polytope):
+        # Translate hyperplanes
+        polyreg.b = polyreg.b + np.dot(polyreg.A, d)
+    else:
+        # Translate subregions
+        for poly in polyreg.list_poly:
+            _translate(poly, d)
+
+    # Translate bbox and cheby
+    if polyreg.bbox is not None:
+        polyreg.bbox = (polyreg.bbox[0] + d,
+                        polyreg.bbox[1] + d)
+    if polyreg._chebXc is not None:
+        polyreg._chebXc = polyreg._chebXc + d
+
+
+def _rotate(polyreg, u=None, v=None, theta=None, R=None):
+    """Rotate this polyreg in place. Return the rotation matrix.
+
+    Simple rotations, by definition, occur only in 2 of the N dimensions; the
```

Comment:
I see that my guess [above](https://github.com/tulip-control/polytope/pull/39/files#r99052275) about what "simple rotation" means was off. Function `_rotate` is hidden, so some (minimal) mention in the docstring of method `Polytope.rotation` of what kind of rotations are supported would help a user.